spec/acceptance/nodesets/debian-610-x64.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  debian-610-x64:
+    roles:
+      - master
+    platform: debian-6-amd64
+    box: puppetlabs/debian-6.0.10-64-nocm
+    box_url: https://vagrantcloud.com/puppetlabs/boxes/debian-6.0.10-64-nocm
```

Comment:
It's just to be thorough. There was already one for Debian 6 but I had trouble with it, so I added this to fix it. I'm okay with removing.